common/scala/src/main/scala/whisk/core/WhiskConfig.scala

```diff
@@ -158,6 +159,7 @@ object WhiskConfig {
     val mainDockerEndpoint = "main.docker.endpoint"
     val routerDockerEndpoint = "router.docker.endpoint"
     val elkDockerEndpoint = "elk.docker.endpoint"
+    val catalogDockerEndpoint = "catalog.docker.endpoint"
```

Comment:
a more general way perhaps of allow other properties is to provide a getter on a string directly.